aten/src/ATen/native/mkldnn/xpu/detail/Attr.h

```diff
@@ -362,4 +365,45 @@ class Attr {
   dnnl::post_ops dnnl_post_ops_;
 };
 
+static inline void construct_attr_by_post_op(
+    const c10::string_view& binary_post_op,
+    double binary_alpha,
+    double input1_scale,
+    int64_t input1_zero_point,
+    // input1_desc,
+    const c10::string_view& unary_post_op,
+    const torch::List<std::optional<at::Scalar>>& unary_post_op_args,
+    const c10::string_view& unary_post_op_algorithm,
+    at::native::onednn::Attr& attr
+){
+    if(binary_post_op == "none"){
```

Comment:
Please use early return, like this:
```cpp
if (binary_pos_op != "none")
    return;
if (unary_post_op == "relu") {
...} elif (...) {
...
}
```

Reply:
hi @guangyey , this if-else branch should have no early return logic. We firstly check binary_post_op value and then check unary_post_op value.

Reply:
Could you pls. refine the code style?